upgrade_deadlines/repair.py

```diff
@@ -25,20 +25,50 @@
 LMS_PASSWORD = os.environ.get('LMS_PASSWORD')
 
 
+def get_access_token():
+    response = requests.post(
+        f'{OAUTH_ACCESS_TOKEN_URL}/access_token',
+        data={
+            'grant_type': 'client_credentials',
+            'client_id': OAUTH_KEY,
+            'client_secret': OAUTH_SECRET,
+            'token_type': 'jwt',
+        },
+        headers={
+            'User-Agent': 'ecommerce-scripts',
+        },
+        timeout=(3.1, 5)
+    )
+
+    try:
+        data = response.json()
+        access_token = data['access_token']
+    except (KeyError, json.decoder.JSONDecodeError) as json_error:
+        logger.exception('Failed to get access token from response.')
+        raise requests.RequestException(response=response) from json_error
+    return access_token
+
+
+def get_course_runs(jwt, querystring):
+    headers = {'Authorization': "JWT {}".format(jwt)}
+    response = requests.get(
+        f'{DISCOVERY_API_URL}course_runs/',
```

Comment:
Same URL comment as the other ones 🙂

Reply:
And for OAUTH_ACCESS_TOKEN_URL: 
https://github.com/openedx/ecommerce-scripts/pull/186/files#diff-9f620f4b04a54f97b94943e5882d47c7e41892f3e1c791571dd2d1b055e0194cR21
Maybe it's worth bringing all the urls to a single format?